Mobile Buy SDK/Mobile Buy SDK/Data/BUYClient+Checkout.m

```diff
@@ -152,6 +153,15 @@ - (NSOperation *)getCompletionStatusOfCheckoutWithToken:(NSString *)token start:
 	}];
 }
 
+- (NSOperation *)pollCompletionStatusAndGetCheckoutWithToken:(NSString *)token start:(BOOL)start completion:(BUYDataCheckoutBlock)block
```

Comment:
Since we really don't want to "leak" details about the underlying implementation in the method signature, perhaps `-getCompletedCheckoutWithToken:…` would be nicer.